File: python/soma_vetor.py
#Problema soma_vetor

N = int(input("Quantos números você vai digitar? "))

vetor: [float] = [0 for x in range(N)]

for i in range(0, N):
    vetor[i] = float(input("Digite um número: "))

print("VALORES = ", end="")
for i in range(0, N):
    print(f"{vetor[i]:.1f}", " ", end="")

soma = 0 
print()
for i in range(0, N):
    soma = soma + vetor[i]
print(f"SOMA = {soma:.2f}")

media = soma / N
print(f"MEDIA = {media:.2f}")

File: java/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Quantos números você vai digitar? ");
		int N = sc.nextInt();
		
		double[] vet = new double[N];
		
		for (int i = 0; i < N; i++) {
			System.out.print("Digite um número: ");
			vet[i] = sc.nextDouble();
		}
		
		System.out.print("VALORES = ");
		for (int i = 0; i < N; i++) {
			System.out.print(String.format("%.2f", vet[i]) + " "); 
		}

		double soma = 0;
		
		System.out.println();
		for (int i = 0; i < N; i++) {
			soma = soma + vet[i];
		}
		System.out.println("SOMA = " + String.format("%.2f", soma));
		
		double media = 0;
		
		media = soma / N;
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		sc.close();
	}

}
